contracts/Colony.sol

```diff
@@ -61,6 +61,23 @@ contract Colony is ColonyStorage {
     });
   }
 
+  function bootstrapColony(uint _amountPerUser, address[] _users) public
+  auth
+  isInBootstrapPhase
+  {
+    colonyBootstrapped = true;
```

Comment:
I don't see a need for this to be its own variable. We can simply say that `taskCount==0` means that we're in the bootstrap phase. Note that this doesn't mean that we replace this line with `taskCount+=1`. There is no inherent problem with calling `bootstrapColony` multiple times (as it is, at least). Indeed, if someone wanted to give *a lot* of people starting reputation and balances, they might need to call the function multiple times if the list was so long that the transaction that contained all of them required more `gas` than the block limit!

Once the first task is created as normal, this function becomes unable to be called because of the condition in `isInBootstrapPhase`.